scripts/build_wheel.py

```
@@ -71,7 +71,10 @@ def clear_folder(folder_path):
         if os.path.isdir(item_path) and not os.path.islink(item_path):
             rmtree(item_path)
         else:
-            os.remove(item_path)
+            try:
+                os.remove(item_path)
+            except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)